packages/venia-ui/lib/components/CartPage/PriceSummary/giftCardSummary.js

```diff
@@ -84,6 +89,7 @@ export const GiftCardSummaryFragment = IS_EE
       `
     : gql`
           fragment GiftCardSummaryFragment on Cart {
+              id
```

Comment:
It wasn't working. I keep seeing Apollo errors about unknown fragment because the fragment definition was missing from the query. Apparently, as I mentioned `__typename` is not enough. When I added `id`, it started working. Without a key in the fragment other than __typename, this is not working.

Reply:
Haha so for some reason I was seeing the error still after checking out your PR. Turns out I didn't fetch the right PR :P Nice find.